features/clip-path-animatable.yml

```
@@ -0,0 +1,6 @@
+name: Animatable clipping paths
+description: Clipping paths in CSS are animated using CSS transitions and animations.
```

Might be nice to make this more concrete:

```suggestion
description: The `clip-path` property can be animated using CSS transitions and animations.
```